demos/TV/TV/MainViewController.m

```diff
@@ -0,0 +1,22 @@
+#import "MainViewController.h"
+
+@implementation MainViewController {
+  UILabel *_label;
+}
+
+- (void)loadView {
+  [super loadView];
+
+  _label = [[UILabel alloc] initWithFrame:CGRectZero];
+  _label.text = @"This project is used to verify that Material Components iOS runs on tvOS";
+  [self.view addSubview:_label];
+}
+
+- (void)viewWillLayoutSubviews {
+  [super viewWillLayoutSubviews];
+
+  [_label sizeToFit];
+  _label.center = CGPointMake(self.view.frame.size.width / 2, self.view.frame.size.height / 2);
```

Comment:
nit: In order to support non-normalized rect's, I recommend the following for getting the center:
CPPoint center = CGPointMake(CGRectGetMidX(self.view.bounds), CGRectGetMidY(self.view.bounds));

Since you are a subview, you typically want to prefer superview.bounds over superview.frame.